test-records.js

```diff
@@ -112,6 +113,56 @@ validator('companies', cdb_schema, (json) => {
                 msg: "Record has `quality` of `tested` but doesn't specify `required-elements`. See: https://github.com/datenanfragen/data/issues/811",
             });
     }
+    // whitespace check
+    Object.keys(json).forEach((key) => {
+        if (typeof json[key] === 'string' && json[key] !== json[key].trim())
+            errors.push(`Seems like \`${key}\` isn't trimmed, i.e. it contains leading or trailing whitespace.`);
+    });
+
+    //// ADDRESS FORMATTING
+    const address_lines = json['address'].split('\n');
+    if (address_lines.length < 2) errors.push('`address` is not formatted with newlines (\\n).');
+    for (line of address_lines) {
+        if (line !== line.trim()) {
+            ("`address` isn't trimmed, i.e. contains unnecessary whitespace.");
+        }
+    }
+    if (address_lines[0].trim() === json['name']) errors.push(' Record includes `name` in the `address`.');
+
+    // check if the last line is a country
+    // this test might produce false-positives, as it's a difficult thing to do
+    // we might consider more fuzzy matching if we get many reports of false positives
+    let last_line_is_country = false;
+    const last_line = address_lines[address_lines.length - 1].trim();
+    for (const [countrycode, v] of Object.entries(countries)) {
+        if ([countrycode, v.name, v.native].includes(last_line)) {
+            last_line_is_country = true;
+            break;
+        }
+        // check for substring match (Netherlands vs *The *Netherlands)
+        if (
+            [countrycode, v.name, v.native].some((x) => {
+                return last_line.includes(x) || x.includes(last_line);
+            })
+        ) {
+            last_line_is_country = true;
+        }
+    }
+    // TODO: replace England with United Kingdom in the DB
+    if (['United States of America', 'England']) last_line_is_country = true;
```

Comment:
I haven't checked the package, but we should only allow once representation for the USA. Personally, I prefer `United States of America` but I'm fine with either as long as we standardize on one.